### PR TITLE
pre57: cleanup, i guess

### PR DESCRIPTION
**Pull request by VillainsRule**
remvoes various unused/annoying shit
also reorders gm for my ocd

### CODE REVIEW

**Comment by onlypuppy7** [comment]
no

File: StateFarmClient.js [hunk 1]
```diff
@@ -94,8 +98,15 @@
 // require:"https://cdnjs.cloudflare.com/ajax/libs/jszip/3.7.1/jszip.min.js"
 // {{!CRACKEDSHELL}}
 
+GM_webRequest([
+    { selector: '*static.hotjar.com/*', action: 'cancel' },
+    { selector: '*adinplay.com/*', action: 'cancel' },
+    { selector: '*btloader.com/*', action: 'cancel' },
+    { selector: '*connect.facebook.net/*', action: 'cancel' },
+    { selector: '*googleadservices.com/*', action: 'cancel' }
+], () => {});
```
Comment (onlypuppy7) [inline StateFarmClient.js:107]: this cant be reverted if the user decides against it/it trips some blocking mechanism

File: StateFarmClient.js [hunk 2]
```diff
@@ -735,7 +734,6 @@ sniping and someone sneaks up on you
             initModule({ location: tp.renderTab.pages[0], title: "Tracers", storeAs: "tracers", bindLocation: tp.renderTab.pages[1], });
             initModule({ location: tp.renderTab.pages[0], title: "Chams", storeAs: "chams", bindLocation: tp.renderTab.pages[1], });
             initModule({ location: tp.renderTab.pages[0], title: "Nametags", storeAs: "nametags", bindLocation: tp.renderTab.pages[1], });
-            initModule({ location: tp.renderTab.pages[0], title: "Targets", storeAs: "targets", bindLocation: tp.renderTab.pages[1], });
```
Comment (onlypuppy7) [inline StateFarmClient.js:738]: no point in removing random shit

File: StateFarmClient.js (hunk 1 above)
Comment (VillainsRule) [reply StateFarmClient.js:107]: it doesn't trip; they can always edit code

File: StateFarmClient.js (hunk 2 above)
Comment (VillainsRule) [reply StateFarmClient.js:738]: it was unused and hence made shit more confusing

**Comment by enbyte** [comment]
old and dead

**Pull request closed without merging**